pkg/api/script.go

```diff
@@ -12,6 +12,17 @@ const (
 	SaveArtifacts Script = "save-artifacts"
 	// Usage i the name of the script responsible for printing the builder image's short info.
 	Usage Script = "usage"
+
```

Comment:
For clarity, it may be useful to separate these constants in 2 blocks... one with 'Script' type constants and the other one for paths.